master/internal/rm/kubernetesrm/pods.go

```diff
@@ -798,7 +824,11 @@ func (p *pods) startEventListeners() error {
 }
 
 func (p *pods) startPreemptionListeners() error {
-	for namespace := range p.namespaceToPoolName {
+	ns, err := workspace.GetAllNamespacesForRM(context.TODO(), p.clusterName, p.namespace)
```

Comment:
nit: context.TODO() should be context.Background(), if it's not passed in the function -- i know there are other context.TODO()s in this function, but that's generally bad code hygiene
(I know stack overflow isn't the truth, but here's a good explanation: https://stackoverflow.com/a/74239191)

Reply:
Got it! Fixed